docs/src/literate/computational_homogenization.jl

```diff
@@ -0,0 +1,548 @@
+    end
+end;
+
+# Just another way to compute the stiffness for testing purposes               #src
+function homogenize_test(u::Matrix, dh, cv, E_incl, E_mat)                     #src
+    ĒΩ = zero(SymmetricTensor{4,2})                                            #src
+    Ω = 0.0                                                                    #src
+    ue = zeros(ndofs_per_cell(dh), 3)                                          #src
+    for cell in CellIterator(dh)                                               #src
+        reinit!(cv, cell)                                                      #src
+        for (localdof, globaldof) in enumerate(celldofs(cell))                 #src
+            for i in 1:3                                                       #src
+                ue[localdof, i] = u[globaldof, i]                              #src
+            end                                                                #src
+        end                                                                    #src
+        E = cellid(cell) in getcellset(dh.grid, "inclusions") ? E_incl : E_mat #src
+        for qp in 1:getnquadpoints(cv)                                         #src
+            dΩ = getdetJdV(cv, qp)                                             #src
+            Ω += dΩ                                                            #src
+            #compute u^ij and u^kl                                             #src
+            Ē′ = SymmetricTensor{4,2}((i, j, k, l) -> begin                    #src
+                #for l in 1:2, k in l:2, j in 1:2, i in j:2                    #src
+                ij = i == j == 1 ? 1 : i == j == 2 ? 2 : 3                     #src
+                kl = k == l == 1 ? 1 : k == l == 2 ? 2 : 3                     #src
+                εij = function_symmetric_gradient(cv, qp, view(ue, :, ij)) +   #src
+                          symmetric((basevec(Vec{2}, i) ⊗ basevec(Vec{2}, j))) #src
+                εkl = function_symmetric_gradient(cv, qp, view(ue, :, kl)) +   #src
+                          symmetric((basevec(Vec{2}, k) ⊗ basevec(Vec{2}, l))) #src
+                return (εij ⊡ E ⊡ εkl) * dΩ                                    #src
+            end)                                                               #src
+            ĒΩ += Ē′                                                           #src
+        end                                                                    #src
+    end                                                                        #src
+    return ĒΩ / Ω                                                              #src
+end                                                                            #src
+
+@test homogenize_test(reduce(hcat, u.dirichlet), dh, cellvalues, 10*C, C) ≈ E_dirichlet #src
+@test homogenize_test(reduce(hcat, u.periodic), dh, cellvalues, 10*C, C) ≈ E_periodic #src
+
+#md # ## [Raw source](@id homogenization-raw-code)
```

Comment:
```suggestion
#md # ## [Plain program](@id homogenization-raw-code)
```
to be consistent with the other examples?

Reply:
I copied this verbatim from some other example, so I think we use different, but I like Plain program better.